ansible_builder/main.py

```diff
@@ -185,6 +186,8 @@ def create_folder_copy_files(self):
             requirement_path = self.definition.get_dep_abs_path(item)
             if requirement_path is None:
                 continue
+            if item == 'python':
+                continue  # will be put into combined requirements.txt
```

Comment:
I _think_ my preference here would be to, instead, read the file and save it to an instance property. Basically, move the logic below at `with open(self.definition.get_dep_abs_path('python'), 'r') as f:` to here.

But... I don't want to be too specific about it right now. Since this is working, I somewhat prefer to finish the processing of collection `bindep.txt` files, see how that code change looks, and change some of the processing order once we have a better sense of that flow.